reference/UQ Handbook Chapter.ipynb

```diff
@@ -20,7 +20,7 @@
    "cell_type": "markdown",
```

Comment:
Can't quite comment in the right spots.

Are the readers going to know what MCMC is? The first couple of sentences seem to be written as if the readers will be fairly familiar with it.